src/videojs-http-streaming.js

```diff
@@ -569,6 +632,15 @@ class HlsHandler extends Component {
     this.tech_.one('canplay',
       this.masterPlaylistController_.setupFirstPlay.bind(this.masterPlaylistController_));
 
+    this.tech_.on('bandwidthupdate', () => {
+      if (this.options_.useBandwidthFromLocalStorage) {
```

Comment:
Might want to add an explicit default value for `useBandwidthFromLocalStorage` in `setOptions_`

Reply:
Any specific reason to set an explicit default value?

Reply:
Just for code as documentation

Reply:
It would be nice to see all option defaults live in an object somewhere